lemur/certificates/models.py

```diff
@@ -62,7 +62,6 @@ def get_sequence(name):
 
 
 def get_or_increase_name(name):
-    name = '-'.join(name.strip().split(' '))
```

Comment:
It seems the only purpose of this line was `name = name.replace(' ', '-')`?

Reply:
Agreed.